README.md

```diff
@@ -310,6 +310,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [GNUnet](https://gnunet.org/) - Free software framework for decentralized, peer-to-peer networking. ([Source Code](https://gnunet.org/git/)) `GPL-3.0` `C`
 - [Gotify](https://gotify.net/) - Self-hosted notification server with Android and CLI clients, similar to PushBullet. ([Source Code](https://github.com/gotify/server), [Clients](https://github.com/gotify/android)) `MIT` `Go`
 - [Hawkpost](https://hawkpost.co) - HawkPost is a web app that lets you create unique links that you can share with a person that desires to send you important information but doesn't know how to encrypt it. The message is encrypted in their browser and sent to your email address. ([Source Code](https://github.com/whitesmith/hawkpost)) `MIT` `Python`
+- [Jam](https://jamshelf.com/) - Jam is an open source alternative to Clubhouse, Twitter Spaces and similar audio spaces. ([Demo](https://jam.systems/), [Source Code](https://github.com/jam-systems/jam)) `AGPL-3.0` `Javascript`
```

Comment:
- the description should explain what the software does (audio conference system?)
- `Javascript` -> `Docker/Nodejs` (https://github.com/jam-systems/jam#install)